setup.cfg

```diff
@@ -25,12 +25,12 @@ data_files =
 	pyraf/clcache = data/clcache/*
 scripts = scripts/pyraf
 
-[extension=pyraf.sscanfmodule]
+[extension=pyraf.sscanf]
```

Comment:
@pllim doesn't this break the import of sscanf in pyraf in Py2 ? (same question for xutil)

It wasn't clear to me where to find the link to the passing Py2-based tests in the "Conversation" tab of this PR.

Reply:
Chris, I'll write up on how to check the Jenkins results as a new issue, so others might benefit from the information.

Reply:
p.s. Same conclusion for `xutil`.